src/htm/algorithms/TemporalMemory.hpp

```diff
@@ -113,11 +113,11 @@ class TemporalMemory : public Serializable
    * duplicates. Disable this for a small speed boost.
    * DEPRECATED: The SDR class now enforces these properties.
    *
-   * @param extra
+   * @param externalPredictiveInputs
    * Number of external predictive inputs.  These inputs are used in addition to
    * the cells which are part of this TemporalMemory.  The TemporalMemory
    * requires all external inputs be identified by an index in the
-   * range [0, extra).
```

Comment:
maybe add? : 
a) If specified, `compute()` will expect additional fied with values in range [0, extraPred) for each iteration. 
b) These values are not related to the current TM, they represent input from a different region.

Reply:
done